<?xml version='1.0'?>
<ui version="4.0">
 <class>ModelsDisplayInfo</class>
 <widget class="QDockWidget" name="ModelsDisplayInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>111</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Sim Models</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>398</width>
         <height>343</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="modelGroup">
          <property name="title">
           <string> ON / OFF </string>
          </property>
          <layout class="QGridLayout" name="gridLayoutNew" columnstretch="0,0,0" columnminimumwidth="0,0,0">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="gGradient_1">
             <property name="text">
              <string>Gravity Gradient Torque</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="rwJitter_1">
             <property name="text">
              <string>Reaction Wheel Jitter</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="gPerturb_1">
             <property name="text">
              <string>Gravity Perturbation</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>+
           <item row="3" column="0">
            <widget class="QLabel" name="atmDrag_1">
             <property name="text">
              <string>Atmosferic Drag</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="srp_1">
             <property name="text">
              <string>Solar Rad Pressure</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="modelGroup2">
          <property name="title">
           <string>Selected Models</string>
          </property>
          <layout class="QGridLayout" name="gridLayoutNew2" columnstretch="0,0,0" columnminimumwidth="0,0,0">
           <item row="6" column="0">
            <widget class="QLabel" name="albedo">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="earthMagField">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionEnabled_Models">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enabled Models</string>
   </property>
  </action>
  <action name="actionSelec_Models">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selected Models</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionEnabled_Models</sender>
   <signal>toggled(bool)</signal>
   <receiver>modelGroup</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
  <connection>
   <sender>actionSelec_Models</sender>
   <signal>toggled(bool)</signal>
   <receiver>modelGroup2</receiver>
   <slot>setVisible(bool)</slot>
  </connection>
 </connections>
</ui>
